sync/statesync/code_queue_test.go

```diff
@@ -96,13 +96,13 @@ func TestCodeQueue(t *testing.T) {
 			recvDone := make(chan struct{})
 			go func() {
 				for _, add := range tt.addCode {
-					require.NoErrorf(t, q.AddCode(add), "%T.AddCode(%v)", q, add)
+					require.NoErrorf(t, q.AddCode(t.Context(), add), "%T.AddCode(%v)", q, add)
 				}
 
 				if tt.quitInsteadOfFinalize {
 					close(quit)
 					<-recvDone
-					require.ErrorIsf(t, q.AddCode(tt.addCodeAfter), errFailedToAddCodeHashesToQueue, "%T.AddCode() after `quit` channel closed", q)
+					require.ErrorIsf(t, q.AddCode(t.Context(), tt.addCodeAfter), errFailedToAddCodeHashesToQueue, "%T.AddCode() after `quit` channel closed", q)
```

Comment:
Our convention is to return the err first for `ErrorIs`
```suggestion
					err := q.AddCode(t.Context(), tt.addCodeAfter)
					require.ErrorIsf(t, err, errFailedToAddCodeHashesToQueue, "%T.AddCode() after `quit` channel closed", q)
```
Not sure why it wasn't caught by the linter though ...